README.md

```diff
@@ -18,7 +18,7 @@ Before using this playbook, ensure the following requirements are met:
 3. **Python**: Python 3.x installed on the control node and all target hosts.
 4. **Privileges**: The user running the playbook must have sudo privileges on the target machines.
 
-**Note**: The following ansible playbook dynamically fetches private validator and node keys from hashicorp vault. 
+**Note**: The following Ansible playbook dynamically retrieves private validator key and node key from HashiCorp Vault. It follows a structured path format: `environment/project/organization/type/file_name` (e.g., `testnet/berachian/encapsulate/validator/priv_validator_key.json`) to fetch the required secrets.
```

Comment:
Can we add a prerequisite section, saying harshicorp vault is used to store and fetch validator keys, where we can give an external link to setup hashicorp vault